commitlint.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
         'concatjs',
         'create',
         'cypress',
+        'esbuild',
```

Comment:
I have this in its own commit in #2476 if we feel like merging that first 🤷

Reply:
Yup can rebase